deployments/green/deployment.yml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: demoK8-green
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: demoK8-v1
+      track: green
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  minReadySeconds: 5
+  template:
+    metadata:
+      labels:
+        app: demoK8-v1
+        track: green
+    spec:
+      containers:
+      - name: demoK8
+        image: gcr.io/PROJECT_ID/IMAGE:TAG
+        ports:
+        - containerPort: 8087
+        readinessProbe:
+          httpGet:
+             path: /actuator/health
+             port: 8087
+          failureThreshold: 6
+          initialDelaySeconds: 120
```

Comment:
decrease initialdelayseconds, the app does not have much functionality. lets keep it at 15 sec